traffic_stops/routers.py

```diff
@@ -31,24 +31,26 @@ def db_for_write(self, model, **hints):
         logger.debug('db_for_write({}): {}'.format(state_db, name))
         return name
 
-    def allow_migrate(self, db, model):
+    def allow_migrate(self, db, app_label, model_name=None, **hints):
         # scenarios:
         #
         # default           traffic_stops_nc    False
         # default           traffic_stops_admin True
         # traffic_stops_nc  traffic_stops_admin False
         # traffic_stops_nc  traffic_stops_nc    True
         #
-        name = self._db_name(model)
-        if db == 'default':
-            if name in settings.DATABASES:
+        db_state = db[-2:]
+        app_is_state = app_label in ('nc', 'md')
```

Comment:
I think the logic can be simplified a lot and at the same can avoid hard-coding the current set of states by following the other methods in the router by assuming that if ``traffic_stops_<applabel>`` is in ``DATABASES`` then ``applabel`` is a state.

The key part would be
```
state_db = self._db_name_from_label(app_label)
app_is_state = state_db in settings.DATABASES
```

(`db_name_from_label()` would be similar to the existing `_db_name()`)